docs/builders/builder-api.md

```diff
@@ -155,7 +155,7 @@ While running in development mode, the builder's development server must be able
 
 ### More information
 
-This area is under rapid development, so the documented is still in progress and subject to change. If you are interested in writing your builder, we encourage you to check [webpack](https://github.com/storybookjs/storybook/tree/next/code/builders/builder-webpack5), [Vite](https://github.com/storybookjs/storybook/tree/next/code/builders/builder-vite), and Modern Web's [dev-server-storybook](https://github.com/modernweb-dev/web/blob/master/packages/dev-server-storybook/src/serve/storybookPlugin.ts) source code. In addition, we have a wonderful contributor community on [Storybook Discord](https://discord.gg/storybook) if you have questions. Ping us in the [#contributing](https://discord.com/channels/486522875931656193/839297503446695956) channel.
+This area is under rapid development, and the associated documentation is still in progress and subject to change. If you are interested in writing your builder, we encourage you to take a look at the source code for [Webpack](https://github.com/storybookjs/storybook/tree/next/code/builders/builder-webpack5), [Vite](https://github.com/storybookjs/storybook/tree/next/code/builders/builder-vite). Modern Web's [dev-server-storybook](https://github.com/modernweb-dev/web/blob/master/packages/dev-server-storybook/src/serve/storybookPlugin.ts) to get a better understanding of how to implement a builder for Storybook, when you are ready open a [RFC](../contribute/RFC.md) to discuss your proposal with the community and the maintainers.
```

Comment:
Hey @jonniebigodes. This second sentence has gotten kinda long. Could we simplify it by breaking it up into two sentences, like so: 

"If you are interested in creating a builder, you can learn more about implementing a builder in Storybook by checking the source code for Vite, Webpack, or Modern Web's dev-server-storybook. When you're ready, open an RFC to discuss your proposal with the Storybook community and maintainers."

(with the links in, as before)